concent_api/core/tests/test_integration_subtask_results_verify.py

```diff
@@ -161,10 +156,7 @@ def test_that_concent_responds_with_http_400_when_verification_received_in_faile
         self._assert_stored_message_counter_increased(increased_by=3)
 
         # when
-        with mock.patch(
-            "core.message_handlers.payments_service.is_account_status_positive",
-            return_value=True
-        ):
+        with mock.patch("core.message_handlers.bankster.claim_deposit", return_value=[True, True]):
```

Comment:
Why don't you use here methods which you create in `utils` as a `return_value`?

Reply:
:+1: done